web/pgadmin/static/js/components/FieldSet.jsx

```diff
@@ -1,30 +1,33 @@
-import { makeStyles } from '@mui/styles';
+import { styled } from '@mui/material/styles';
 import React from 'react';
-import clsx from 'clsx';
 import PropTypes from 'prop-types';
 import CustomPropTypes from '../custom_prop_types';
 
-const useStyles = makeStyles((theme)=>({
-  fieldset: {
+const StyledFieldset = styled('fieldset')((
+  {
+    theme
+  }
+) => ({
+  '&.FieldSet-root': {
     padding: theme.spacing(0.5),
     borderRadius: theme.shape.borderRadius,
     backgroundColor: 'inherit',
     border: '1px solid ' + theme.otherVars.borderColor,
+    '& .FieldSet-legend': {
+      width: 'unset',
+      fontSize: 'inherit',
+      fontWeight: 'bold',
+    }
   },
-  legend: {
-    width: 'unset',
-    fontSize: 'inherit',
-    fontWeight: 'bold',
-  }
 }));
 
 export default function FieldSet({title='', className, children}) {
-  const classes = useStyles();
+
   return (
-    <fieldset className={clsx(classes.fieldset, className)}>
-      <legend className={classes.legend}>{title}</legend>
+    <StyledFieldset className={'FieldSet-root ' +  className}>
```

Comment:
no need to add class to root if it is not conditional

Reply:
Done.